src/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoData.cs

```diff
@@ -29,23 +29,19 @@ public static string JaJPAbbreviatedEraName()
 
         public static string[] FrFRDayNames()
         {
-#if !uap
```

Comment:
Do you know why these were here previously?  They're surrounding tests guarded by macOS checks and thus wouldn't run on uap anyway.

Reply:
I assume that the PlatformDetection.IsOSX checks used PInvokes which crashed during runtime on UAP. This is just a guess.